scripts/wdio/wdio-runner.js

```diff
@@ -0,0 +1,49 @@
+const Launcher = require('webdriverio').Launcher;
+
+async function wdioRunner(options) {
+  const {
+    configPath, locales, formFactors, useSeleniumGrid, continueOnFail, ...testSetup
+  } = options;
+  const factors = formFactors || [undefined];
+
+  process.env.USE_SELENIUM_GRID = useSeleniumGrid;
```

Comment:
Should we keep this separate from this script? It seems like an implementation leak.

`USE_SELENIUM_GRID='true' tt-wdio --config ./wdio.conf.js --locales ['en','es']`
vs.
`tt-wdio --config./wdio.conf.js --locales ['en','es'] -- useSeleniumGrid true`

Reply:
hmmm I see what you mean. I will remove `useSeleniumGrid` script option in favor of passing the env var directly.